docs/secure-connections/sauce-connect/proxy-tunnels.md

```diff
@@ -403,16 +419,29 @@ Ephemeral tunnels (short-lived tunnels) are ideal for the following test situati
 #### Starting an Ephemeral Tunnel From Your Local Workstation
 One option to start Ephemeral tunnels is to do so from your local workstation.
 
-1. Set your Sauce Labs username and access key as environmental variables. For more information, see [Using Environment Variables for Authentication Credentials](/basics/environment-variables).
+1. [Set your Sauce Labs username and access key as environmental variables](/basics/environment-variables).
+2. Run the basic startup commands (i.e., no optional flags) to ensure that the tunnel starts:
```

Comment:
I think we should include the strongly recommended options here. The region flag will make sure they know where to look for their tunnels in the web ui, and the tunnel_name will make sure they don't accidentally have tests running through multiple anonymous tunnels.